### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by skywinder**
fix warnings in example for modern swift syntax

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,20 +41,20 @@ let config = TokenConfiguration(token: "12345")
 or for GitHub Enterprise
 
 ```swift
-let config = TokenConfiguration("12345", url: "https://github.example.com/api/v3/")
+let config = TokenConfiguration("2beef4ace", url: "https://github.example.com/api/v3/")
```
Comment (pietbrauer) [inline README.md]: This seems not needed

File: README.md [hunk 2]
```diff
@@ -41,20 +41,20 @@ let config = TokenConfiguration(token: "12345")
...
 ```swift
 Octokit(config).me() { response in
-  switch response {
-  case .success(let user):
-    print(user.login)
-  case .failure(let error):
-    print(error)
-  }
-}
+      switch response {
```
Comment (pietbrauer) [inline README.md]: Could you fix the indentation again so only the `as Any` is added?

File: README.md (hunk 1 above)
Comment (skywinder) [reply README.md]: just more clear that it's token, not the digits

**Comment by skywinder** [comment]
hmm, I changed it directly on GH, I don't see any options to modify this pr ¯\_(ツ)_/¯

**Comment by skywinder** [comment]
close in favor of #90

**Pull request closed without merging**